templates/openstackclient/bin/init.sh

```diff
@@ -69,3 +60,59 @@ EOF
   git config --global user.email "dev@null.io"
   git config --global user.name "OSP Director Operator"
 fi
+
+
+# Skip the rest on tempest pod
+if cat /etc/hostname | grep ^tempest 2>&1 > /dev/null; then exit 0; fi
+
+if [ "$IPA_SERVER" != "" -a ! -f /var/lib/ipa-client/sysrestore/sysrestore.index ]; then
+
+  # Ensure hostname is correct for ipa-client install
+  sudo bash -c "cat /mnt/etc/hostname > /etc/hostname"
+  # Ensure hostname -f and python socket.getfqdn() return the FQDN
+  SHORT_HOSTNAME=$(hostname -s)
+  LONG_HOSTNAME=$(hostname -f)
+  if [ "$LONG_HOSTNAME" != "$FQDN" ]; then
+    sed -i -e "s/^\([0-9.]\+\)\s\+\(.*\)${SHORT_HOSTNAME}\$/\1\t\2${FQDN} ${SHORT_HOSTNAME}/" /mnt/etc/hosts
+    sudo bash -c "cat /mnt/etc/hosts > /etc/hosts"
+  fi
+
+  cat <<EOF > /home/cloud-admin/openstackclient_ipa_install.yaml
```

Comment:
TODO: move this playbook to the tripleo-ipa repo.

It's based on the undercloud-ipa-install playbook but uses ipa-client-install directly instead of using the freeipa-ansible role (that fails as it tries to use hostnamectl which requires systemd).